internal/discourse/discourse.go

```diff
@@ -0,0 +1,88 @@
+package discourse
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+)
+
+const (
+	internalTestCategory  = 29
+	buildIssuesCat = 11
+)
+
+var (
+	discourseAPIKey     = os.Getenv("DISCOURSE_API_KEY")
+	discourseAPIUser    = os.Getenv("DISCOURSE_API_USER")
+	discourseCategoryID int
+	topicTpl = `Original GitHub post: %s
+	
+	%s`
+)
+
+func init() {
+	if discourseAPIKey == "" {
+		fmt.Println("DISCOURSE_API_KEY empty")
+		os.Exit(1)
+	}
+
+	if discourseAPIUser == "" {
+		fmt.Println("DISCOURSE_API_USER empty")
+		os.Exit(1)
+	}
+
+	flag.IntVar(&discourseCategoryID, "discourse-category-id", internalTestCategory, "--discourse-category-id=<int> (discourse category to post topics to)")
+}
+
+func PostTopic(title string, originURL, content string) (string, error) {
+	message := make(map[string]interface{})
+
+	message["title"] = title
+	message["category"] = discourseCategoryID
+	message["raw"] = fmt.Sprintf(topicTpl, originURL, content)
+
+	payload, err := json.Marshal(message)
+	if err != nil {
+		return "", fmt.Errorf("could not marshal %s; reason: %s", message, err)
+	}
+
+	url := fmt.Sprintf("https://discuss.bitrise.io/posts.json?api_key=%s&api_username=%s", discourseAPIKey, discourseAPIUser)
+	resp, err := http.Post(url, "application/json", bytes.NewBuffer(payload))
+	if err != nil {
+		return "", fmt.Errorf("error posting payload %s: %s", payload, err)
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	defer func() {
+		if err := resp.Body.Close(); err != nil {
+			fmt.Printf("warning: could not close response body: %s", err)
```

Comment:
I think it would be better to use our [log package](https://github.com/bitrise-io/go-utils/tree/master/log)
And write this in a warning instead of the default printer.